python/tchannel/tornado/connection.py

```diff
@@ -405,18 +394,10 @@ def serve(self, handler):
         assert handler, "handler is required"
         assert self._loop_running, "Finish the handshake first"
 
+        connection = conn or self
```

Comment:
If you need to do this, might as well make this a class method and always pass in conn